community/people/gaeg/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: person
+title: Gerson Adimir Esquivel Garcia
+role: Undergraduate Student
+image: Gerson-3 copy.jpg
+email: esquivelgarc@wisc.edu
+phone: "608/4194316"
+city_state_zip: "Madison, WI 53706"
+services:
+  github: GersonEsquivel
+  facebook: Gerson Esquivel Garcia
```

Comment:
I think this should be the value for direct access to your page via a facebook url:
```suggestion
  facebook: gerson.esquivel.garcia
```

Reply:
Fixed